app/models/generation.rb

```diff
@@ -0,0 +1,49 @@
+# generation.csv displays database
+# i.e. all groups that should have a generation-value must be included. For groups that are not included generation is going to be set nil
+
+class Groups < ActiveRecord::Base
+
+    def self.import_from_csv(file)
+        data = Buzzn::Utils::File.read(file)
+        check = []
+
+        CSV.parse(data.gsub(/\r\n?/, "\n"), col_sep: ';', headers: true) do |row|
+            gen = row['Generation']
+            group = Group::Localpool.find(Hash[row.to_a]['ID'])
```

Comment:
In dieser Zeile wird eine ActiveRecord::RecordNotFound Exception geworfen, falls die Gruppe mit der id Hash[row.to_a]['ID'] nicht existiert. Diese Exception sollte mit einem begin...rescue Block abgefangen werden. Diese könnte z.b. in etwa so aussehen:
```
begin
                group = Group::Localpool.find(Hash[row.to_a]['ID'])
rescue ActiveRecord::RecordNotFound => e
                logger.error("Group with id #{Hash[row.to_a]['ID']} not found")
end
```